codes/1965_cpp_dbstn322.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+
+using namespace std;
+int dp[1001];
+
+int main()
+{
+    int n;
+    cin >> n;
+
+    int box[n];
+
+    for (int i = 0; i < n; i++)
+    {
+        int num;
+        cin >> num;
+        box[i] = num;
+    }
+
+    int max = 0;
+
+    for (int i = 0; i < n; i++)
+    {
+        dp[i] = 1;
+        for (int j = 0; j < i; j++)
+        {
+            if (box[j] < box[i] && dp[j] >= dp[i])
+                dp[i] = dp[j] + 1;
+        }
```

Comment:
만약 algorithm 헤더파일을 인클루드해서 max 함수를 쓴다면 다음과 같이 간략하게 쓸 수 있습니다.
```suggestion
            if (box[j] < box[i])
                dp[i] = max(dp[i], dp[j] + 1);
```

Reply:
넵 반영하겠습니다! 더 간단해지네요